sources/packages/backend/apps/queue-consumers/src/main.ts

```diff
@@ -0,0 +1,47 @@
+require("../../../env_setup_apps");
+import { Queues } from "@sims/services/queue";
+import { ConfigService } from "@sims/utilities/config";
+import { createBullBoard } from "@bull-board/api";
+import { BullAdapter } from "@bull-board/api/bullAdapter";
+import { ExpressAdapter } from "@bull-board/express";
+import { NestFactory } from "@nestjs/core";
+import { Queue } from "bull";
+import { QueueConsumersModule } from "./queue-consumers.module";
+import * as basicAuth from "express-basic-auth";
+
+async function bootstrap() {
+  const app = await NestFactory.create(QueueConsumersModule);
+
+  const config = app.get<ConfigService>(ConfigService);
+
+  //Application port.
+  const port = config.queueConsumersPort || 3001;
```

Comment:
can't we add `|| 3001` in queueConsumersPort() config service itself?  something like this
```
get queueConsumersPort(): number {
    return this.getCachedConfig(
      "queueConsumersPortConfig",
      +process.env.QUEUE_CONSUMERS_PORT || 3001,
    );
  }
```

Reply:
Just done in previous commit.